core/src/main/resources/config-spring-geonetwork.xml

```diff
@@ -187,4 +187,22 @@
   </bean>
 
 
+  <bean id="filesystemStore"
+        class="org.fao.geonet.api.records.attachments.FilesystemStore"/>
+  <!-- If you want data stored in S3, use that instead: -->
+  <!--  <bean id="filesystemStore" class="org.fao.geonet.api.records.attachments.S3Store">-->
+  <!--    <property name="region" value="eu-west-1"/>-->
+  <!--    <property name="bucket" value="geonetwork-test"/>-->
+  <!--    <property name="keyPrefix" value="geonetwork"/>-->
+  <!--    &lt;!&ndash; Only needed if you don't have a ~/.aws/credentials &ndash;&gt;-->
```

Comment:
is the comment markup encoding on purpose?

Reply:
When I ask my editor to comment out a section of XML, this is what it does. It's a comment that would messup the comment surrounding it.